### PR TITLE
CUMULUS-2959:Convert numeric granule.productVolume to string

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-2959: Remove from CMR failed on schema validation for records prior to release 11](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2959)

## Changes

* After a granule is removed from CMR, the granule record needs to be updated in the database.  Before the fix, the update of an old record (with numeric productVolume) will fail.  After the fix, the productVolume will be converted to a string, and update should work.

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: packages/api/lib/granules.js [hunk 1]
```diff
@@ -33,14 +34,21 @@ const translateGranule = async (
   granule,
```
Comment (Jkovarik) [inline packages/api/lib/granules.js:34]: Not a problem *from* this PR, but there's no unit around this method directly.    This logic needs a doc and unit test.

File: packages/api/tests/models/granules/test-granules-model.js [hunk 2]
```diff
@@ -282,7 +282,7 @@ test('files existing at location returns only file that exists with multiple des
   ]);
 });
 
-test('get() will translate an old-style granule file into the new schema', async (t) => {
+test('get() will translate an old-style granule file and numeric productVolume into the new schema', async (t) => {
```
Comment (Jkovarik) [inline packages/api/tests/models/granules/test-granules-model.js:285]: Given these are in the api model tests, we should make sure that the feature branch granules work is compatible with the updates.   The method being changed is in the api/lib directory (see comment on unit test, though), but these tests may not diretly exist in feature.

File: packages/es-client/tests/test-execution-upsert.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ test.after.always(async (t) => {
   await cleanupTestIndex(t.context);
 });
 
-test.only('upsertExecution writes new "running" execution', async (t) => {
+test.serial('upsertExecution writes new "running" execution', async (t) => {
```
Comment (Jkovarik) [inline packages/es-client/tests/test-execution-upsert.js:34]: Ouch, how did that get in there!

**Review by Jkovarik — CHANGES_REQUESTED** [review]
This looks great Jenny!  Thanks for your patience with the review delay.   Just one minor concern and it should be good to go.     Great work!

File: packages/api/lib/granules.js (hunk 1 above)
Comment (jennyhliu) [reply packages/api/lib/granules.js:34]: added doc and unit test

**Review by Jkovarik — APPROVED** [review]
Thanks @jennyhliu  this looks good.  There's still an outstanding question re: phase-3 feature branch for the ticket, but this looks good to merge.

**Comment by jennyhliu** [comment]
> Thanks @jennyhliu this looks good. There's still an outstanding question re: phase-3 feature branch for the ticket, but this looks good to merge.

rds-phase-3 won't need the fix since DynamoDB is gone, rds returns string product_volume

**Pull request merged**